project.json

```diff
@@ -473,14 +473,14 @@
                     "versions": {
                         "40": [
                             {
-                                "version": "40",
+                                "version": "41",
```

Comment:
This would need to be updated in the docs and `README.md`.

Reply:
https://github.com/vmware-samples/packer-examples-for-vsphere/blob/fb37d5e8488c5f253461c8328e7cad32202d76a5/docs/index.md?plain=1#L37

Reply:
Resolved in https://github.com/vmware-samples/packer-examples-for-vsphere/commit/3cb90f4e13531fc53075a486a3852e6b6cdd4444.